admin/app/webpack/javascripts/components/node-form.es6

```diff
@@ -56,9 +56,29 @@ export default Vue.component('node-form', {
   methods: {
     setSlug: function() {
       if (this.node['new_record?']) {
-        this.node.slug = this.node.title.toLowerCase().replace(/[\s\_]/g, '-').replace(/[^\w\-]/g, '');
+        this.node.slug = this.node.title.toLowerCase();
```

Comment:
Just realized this is all in lowercase so I could dump half of the lines after this line basically. Will do that tonight.